lib/main.dart

```diff
@@ -77,13 +71,17 @@ class HomePageState extends State<HomePage> {
         new Row(
           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
           children: [
-            buildArrowColumn(Icons.arrow_back, () => setState(() {
-              if (_numPlayers > minPlayers) _numPlayers--;
-            })),
+            buildArrowColumn(
+                Icons.arrow_back,
+                () => setState(() {
+                      if (_numPlayers > minPlayers) _numPlayers--;
```

Comment:
ifs with no brackets make me sad buuuuut ok, this is unlikely to ever do anything else